server/test/api/debtorGroups.js

```diff
@@ -0,0 +1,186 @@
+/* global describe, it, beforeEach */
+
+var chai = require('chai');
+var expect = chai.expect;
+
+var helpers = require('./helpers');
+var uuid    = require('node-uuid');
```

Comment:
It's excellent to see you using `node-uuid`.  This is a step in the right direction!